Formula/k3d.rb

```diff
@@ -0,0 +1,19 @@
+class K3d < Formula
+  desc "Little helper to run Rancher Lab's k3s in Docker"
+  homepage "https://github.com/rancher/k3d"
+  url "https://github.com/rancher/k3d/archive/v1.2.1.tar.gz"
+  sha256 "eb7510aab675183b7febbad79ee23e04de6e6e47369c898ccf03ec2022045709"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "install-tools"
+    system "make", "BINDIR='#{bin}'", "GIT_TAG='1.2.1'", "build"
```

Comment:
Please use `#{version}` instead of hardcoding the version.

Reply:
fixed